### DISCUSSION TITLE
O3D303 camera parameter config  C++ on Linux

### DISCUSSION THREAD

**Discussion by cuix1205** [ifm/ifm3d#396] [Q&A]
Now, I have successfully set the O3D303 camera configuration parameters through the terminal command line under Ubuntu, but I want to write these adjustment interfaces into C++
How can I use the program in ifm3d lib in the main program?

**Comment by lola-masson** [comment]
Hi @cuix1205, 
You should be able to use the ifm3d library as a c++ dependency like you would any other, have a look at the basic usage document (for ifm3d 0.20.3 you can find it here: https://github.com/ifm/ifm3d/blob/v0.20.3/doc/basic_usage.md).
